client/src/client.rs

```diff
@@ -1,32 +1,23 @@
 use anyhow::bail;
 use async_channel::Receiver;
+use givc_common::pb;
+pub use givc_common::query::{Event, QueryResult};
 use tokio_stream::StreamExt;
 use tonic::transport::Channel;
 use tracing::debug;
 
 use givc_common::address::EndpointAddress;
-use givc_common::pb;
```

Comment:
I (started) grouped imports intentionally -- std, libraries, project crates, this crates import (internally alphabetically sorted)

Reply:
Ah, yes, this got messed up in a rebase.